javascript/2-functions/hw-1/scripts.js

```diff
@@ -1 +1,20 @@
 // Your JS code is here
+function tecllFortune(numOfKids, partnerName, geoLocation, jobTitle) {
+  var p = document.createElement("p");
+  p.innerHTML =
+    "You will be a " +
+    jobTitle +
+    " in " +
+    geoLocation +
+    ", and married to " +
+    partnerName +
+    " with " +
+    numOfKids +
+    " kids.";
+  document.body.appendChild(p);
+}
```

Comment:
if I'm not mistaking, this class is pure javascript, so I would suggest you to use only javascript. I will approve this time but next time please create   a string instead of an element and use console.log().

Reply:
alright, ty